static/skin/index.js

```diff
@@ -26,7 +26,9 @@
                 filteredParams.append(key, value);
             }
         }
-        document.querySelector('#feed_link').href = `${root}/catalog/v2/entries?${filteredParams.toString()}`;
+        const feedLink = `${root}/catalog/v2/entries?${filteredParams.toString()}`;
+        document.querySelector('#feed_link').href = feedLink;
+        document.querySelector('#feedLink').href = feedLink;
     }
```

Comment:
I see in the current naming of the OPDS feed link ids a risk that any future links will be named `feedlink`, `FeedLink`, etc :)

Reply:
I was worried about it too :P but kelson's recommendation so I let it pass.
What do you think is a better name?
@kelson42 @veloman-yunkan

Reply:
We should keep only version of `feedLink` or `feed_link` to have the naming consistant.
But we should have a id for the link in the head as `head_feed_link` (or `headFeedLink`) and `feed_link` (or `FeedLink`).

Reply:
`headFeedLink` then :)